<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="11" hostname="Mac-mini-DuniceEdCast.local" failures="1" tests="1" name="Command line test" time="6.688" errors="0" timestamp="2021-12-29T14:22:01 MSK">
  <testcase classname="WrongLoginTest" name="userDoesNotExist" time="5.425">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;400&amp;gt; but was &amp;lt;404&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <404>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:92)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at io.restassured.internal.ValidatableResponseImpl.super$2$spec(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.spec(ValidatableResponseImpl.groovy:76)
at io.restassured.internal.ValidatableResponseImpl.spec(ValidatableResponseImpl.groovy)
at WrongLoginTest.userDoesNotExist(WrongLoginTest.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 47 stack frames]]>
    </failure>
  </testcase> <!-- userDoesNotExist -->
  <testcase classname="WrongLoginTest" name="incorrectEmail2" time="0.0">
    <ignored/>
  </testcase> <!-- incorrectEmail2 -->
  <testcase classname="WrongLoginTest" name="emptyEmailAndPassword" time="0.0">
    <ignored/>
  </testcase> <!-- emptyEmailAndPassword -->
  <testcase classname="WrongLoginTest" name="incorrectEmail3" time="0.0">
    <ignored/>
  </testcase> <!-- incorrectEmail3 -->
  <testcase classname="WrongLoginTest" name="emptyEmail" time="0.0">
    <ignored/>
  </testcase> <!-- emptyEmail -->
  <testcase classname="WrongLoginTest" name="incorrectEmail4" time="0.0">
    <ignored/>
  </testcase> <!-- incorrectEmail4 -->
  <testcase classname="WrongLoginTest" name="incorrectEmail1" time="0.0">
    <ignored/>
  </testcase> <!-- incorrectEmail1 -->
  <testcase classname="WrongLoginTest" name="withoutEmailAndPassword" time="0.0">
    <ignored/>
  </testcase> <!-- withoutEmailAndPassword -->
  <testcase classname="WrongLoginTest" name="emptyPassword" time="0.0">
    <ignored/>
  </testcase> <!-- emptyPassword -->
  <testcase classname="WrongLoginTest" name="incorrectEmail5" time="0.0">
    <ignored/>
  </testcase> <!-- incorrectEmail5 -->
  <testcase classname="WrongLoginTest" name="WithoutEmail" time="0.0">
    <ignored/>
  </testcase> <!-- WithoutEmail -->
  <testcase classname="WrongLoginTest" name="correctEmailWithoutPassword" time="0.0">
    <ignored/>
  </testcase> <!-- correctEmailWithoutPassword -->
</testsuite> <!-- Command line test -->
